### PR TITLE
Remove prepublish script

### PR DESCRIPTION
**Pull request by felixfbecker**
Context: https://sourcegraph.slack.com/archives/CMT39K56Z/p1596534092075000

### CODE REVIEW

File: web/gulpfile.js [hunk 1]
```diff
@@ -72,15 +79,17 @@ async function webpackDevelopmentServer() {
 /**
  * Builds everything.
  */
-const build = gulp.parallel(gulp.series(gulp.parallel(schema, graphQlSchema), gulp.parallel(webpack)))
+const build = gulp.parallel(
+  gulp.series(gulp.parallel(schema, graphQlOperations, graphQlSchema), gulp.parallel(webpack))
+)
```
Comment (eseliger) [inline web/gulpfile.js:84]: isn't the parallel of a single series a noop?

Comment (felixfbecker) [reply web/gulpfile.js:84]: Yes, haha

**Comment by felixfbecker** [comment]
> but I think we should document that you will need to run yarn generate once you changed the schema.

You shouldn't have to though as long as `./dev/start` is running (just like before)?

**Comment by eseliger** [comment]
I think a couple of people (including me) develop while not running `dev/start` and just start up as needed. But agree it's no change over the previous behavior, so we can defer that to a separate PR.

**Comment by felixfbecker** [comment]
Yeah I belong to the group too. I just added a VS Code task that you can manually in that case. I think it would be a generally good idea to add docs on our codegen, TypeScript composite projects, and "how to get compile errors" (also not ./dev/start!) to https://github.com/sourcegraph/sourcegraph/blob/master/doc/dev/web/build.md .
I'd like to not block on this though because the PR as is doesn't change this behavior but still improves the experience for backend devs.

**Comment by eseliger** [comment]
Sure, go ahead :)

**Pull request merged**